### PR TITLE
Fix/NSA-106/Update styles for ckeditor v.4.11.1

### PR DESCRIPTION
**Pull request by andrey-ponamarev**
Related to task: https://oat-sa.atlassian.net/browse/NSA-106
Description: CKEditor is [updated](https://github.com/oat-sa/ckeditor-dev/pull/16) and required small changes in styles

### CODE REVIEW

File: scss/ckeditor/skins/tao/scss/dialog.scss [hunk 1]
```diff
@@ -49,6 +49,10 @@ Comments in this file will give more details about each of the above blocks.
   visibility: visible;
 }
 
+.cke_dialog_container {
+  background: transparent!important;
```
Comment (bziondik) [inline scss/ckeditor/skins/tao/scss/dialog.scss:53]: can it be done without `!important`?

File: scss/ckeditor/skins/tao/scss/inc/_ck-icons.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* Do not edit */@import "inc/bootstrap";
+.cke .cke_top .cke_toolbar .cke_button_icon {background-image: none!important;}
```
Comment (bziondik) [inline scss/ckeditor/skins/tao/scss/inc/_ck-icons.scss:2]: can it be done without !important?

File: scss/ckeditor/skins/tao/scss/dialog.scss (hunk 1 above)
Comment (andrey-ponamarev) [reply scss/ckeditor/skins/tao/scss/dialog.scss:53]: Unfortunately not it's overrides styles with `!important` property provided by dialog plugin

File: scss/ckeditor/skins/tao/scss/inc/_ck-icons.scss (hunk 2 above)
Comment (andrey-ponamarev) [reply scss/ckeditor/skins/tao/scss/inc/_ck-icons.scss:2]: Unfortunately not it's overrides styles with `!important` property provided by plugin

**Pull request merged**